cmake/configs/default.cmake

```diff
@@ -51,6 +51,9 @@ hunter_config(CppNetlibUri VERSION 1.0.4-hunter)
 hunter_config(CsvParserCPlusPlus VERSION 1.0.1)
 hunter_config(Eigen VERSION 3.2.4-p0)
 hunter_config(Expat VERSION 2.1.1)
+if(MSVC)
+  hunter_config(getopt VERSION 1.0.0-p0)
```

Comment:
I would like to get getopt, should we change "https://github.com/hunter-packages/wingetopt" to "https://github.com/hunter-packages/getopt"?